scipy/stats/stats.py

```diff
@@ -205,6 +205,9 @@
 
 
 def _chk_asarray(a, axis):
+    if len(np.array(a).shape) == 0:
```

Comment:
Thanks @jaimefrio. I made the adjustment. I am new to the community and trying to become a useful member so I quite value your input.